Calculate/Program.cs

```diff
@@ -0,0 +1,36 @@
+namespace Calculate;
+
+public class Program
+{
+    public Action<string> WriteLine { get; init; } = Console.WriteLine;
+    public Func<string?> ReadLine { get; init; } = Console.ReadLine;
+
+    public static void Main() => new Program().Run();
+
+    public void Run()
+    {
+        Calculator calculator = new();
+        bool run = true;
+
+        while (run)
```

Comment:
Consider that having an infinite loop creates a certain kind of interactive experience like a [REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop) vs if you did not have the loop the app would exit on each invocation and could be used like in an automated tool call fashion.